contracts/payloads/IGP67/PayloadIGP67.sol

```diff
@@ -0,0 +1,458 @@
+            });
+            setDexLimits(DEX_weETHs_ETH); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(getDexAddress(14), TEAM_MULTISIG, true);
+        }
+    }
+
+    /// @notice Action 2: Set dust allowance to rsETH-ETH<>wstETH, rsETH<>wstETH, weETHs-ETH<>wstETH vaults
+    function action2() internal {
+        {
+            // [TYPE 2] rsETH-ETH<>wstETH | Smart collateral & debt
+            Vault memory VAULT_rsETH_ETH_AND_wstETH = Vault({
+                vault: getVaultAddress(78),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 40_000, // $40k
+                maxBorrowLimitInUSD: 50_000 // $50k
+            });
+
+            setVaultLimits(VAULT_rsETH_ETH_AND_wstETH); // TYPE_2 => 78
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(78),
+                TEAM_MULTISIG,
+                true
+            );
+        }
+
+        {
+            // [TYPE 1] rsETH<>wstETH | collateral & debt
+            Vault memory VAULT_rsETH_AND_wstETH = Vault({
+                vault: getVaultAddress(79),
+                vaultType: TYPE.TYPE_1,
+                supplyToken: rsETH_ADDRESS,
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 50_000, // $50k
+                baseBorrowLimitInUSD: 40_000, // $40k
+                maxBorrowLimitInUSD: 50_000 // $50k
+            });
+
+            setVaultLimits(VAULT_rsETH_AND_wstETH); // TYPE_1 => 79
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(79),
+                TEAM_MULTISIG,
+                true
+            );
+        }
+
+        {
+            // [TYPE 2] weETHs-ETH<>wstETH | Smart collateral & debt
+            Vault memory VAULT_weETHs_ETH_AND_wstETH = Vault({
+                vault: getVaultAddress(80),
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 40_000, // $40k
+                maxBorrowLimitInUSD: 50_000 // $50k
+            });
+
+            setVaultLimits(VAULT_weETHs_ETH_AND_wstETH); // TYPE_2 => 80
+
+            VAULT_FACTORY.setVaultAuth(
+                getVaultAddress(80),
+                TEAM_MULTISIG,
+                true
+            );
+        }
+    }
+
+    /// @notice Action 3: Increase Allowance and LTV of stETH redemption protocol
+    function action3() internal {
+        { // Increase Allowance to 10k wstETH
+            uint256 amount_ = getRawAmount(wstETH_ADDRESS, 10_000, 0, false);
+
+            BorrowProtocolConfig memory config_ = BorrowProtocolConfig({
+                protocol: 0x1F6B2bFDd5D1e6AdE7B17027ff5300419a56Ad6b,
+                borrowToken: wstETH_ADDRESS,
```

Comment:
stETH protocol borrows ETH, need to adjust here and above when getting amount & comment

Also let's leave expand settings as they are currently, which is 

"expandPercent": "0",
"expandDuration": "1",